.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Node.js CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test-build-release:
+    env:
+      IS_PUSH_TO_MASTER: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
```

Comment:
Remove, this variable does not appear to be used.

Reply:
Removed in 0e1ef61b87da99a059e8f660e536eec3b83ad8db.